algoliasearch/src/offline/java/com/algolia/search/saas/MirroredIndex.java

```diff
@@ -289,7 +289,7 @@ public void setDelayBetweenSyncs(long delayBetweenSyncs)
      * @return The delay, expressed in <code>unit</code>.
      */
     public long getDelayBetweenSyncs(@NonNull TimeUnit unit) {
-        return unit.convert(this.getDelayBetweenSyncs(), TimeUnit.MILLISECONDS);
+        return TimeUnit.MILLISECONDS.convert(this.getDelayBetweenSyncs(), unit);
```

Comment:
No, this one was correct... 😈 We want to convert the internal delay (expressed in milliseconds) to whatever unit the client asked.

Reply:
Oups